tests/sentry/test_event_consumer.py

```diff
@@ -1,34 +0,0 @@
-from __future__ import absolute_import, print_function
-
-import mock
-
-from sentry.event_consumer import process_event_from_kafka
-from sentry.signals import event_accepted
-from sentry.testutils import (assert_mock_called_once_with_partial, TestCase)
-from sentry.utils import json
-
-
-class EventConsumerTest(TestCase):
-    @mock.patch('sentry.web.api.kafka_publisher')
-    def test_event_consumer(self, mock_kafka_publisher):
-        with self.options({
-            'store.kafka-sample-rate': 1.0,
-            'store.process-in-kafka': True,
-            'kafka-publisher.raw-event-sample-rate': 0.0,
-        }):
-            mock_event_accepted = mock.Mock()
-            event_accepted.connect(mock_event_accepted)
-
-            resp = self._postWithHeader({'logentry': {'message': u'hello'}})
```

Comment:
Didn't realize there was coupling in this test case here, so this also needs to be removed as part of this PR as well for a clean split.